about/changelog.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Changelog
+excerpt: Changelog
+keywords: [changelog, upgrades, updates, releases]
+---
+
+# Changelog
+
+## TimescaleDB 2.15.2
+
+June 7, 2024
+
+🐛 Bugfixes
+
+- [#6975](https://github.com/timescale/timescaledb/issues/6975): Fix sort pushdown for partially compressed chunks.
+- [#6976](https://github.com/timescale/timescaledb/issues/6976): Fix removal of metadata function and the update script.
+- [#6978](https://github.com/timescale/timescaledb/issues/6978): Fix segfault in compress_chunk with primary space partition.
+- [#6993](https://github.com/timescale/timescaledb/issues/6993): Disallow hash partitioning on the primary column.
+
+## May 2024 Update
```

Comment:
But many posts will have content that span multiple products 🤔 That's _why_ I went with a date approach, because this post was a grab bag of _stuff_ (two new extensions and one PopSQL improvement)

Reply:
Check out the titles on this page: https://learn.hex.tech/changelog

- [2024 Spring Release](https://learn.hex.tech/changelog/2024-05-29)
- [Audit logs, expanded Git support, and Magic chart improvements](https://learn.hex.tech/changelog/2024-04-30)
- [Reviews, data browser updates, and more](https://learn.hex.tech/changelog/2024-04-03)

That's the style I was going for. The idea is that it catches the user's attention, like "ohh I'm interested in the git integration, and I can see git is in the title"

Reply:
Hmmm nice point, and I like their doc. 

All Hex releases are all done by date. To be consistent in this page, you need to change the TimescaleDB 2.15.2 header to  "June 7th 2024 update" . I'd keep to specific date, patch releases can come out pretty fast. 

@pallavisontakke, @fabriziomello are you ok with this?

Reply:
Hex is kinda weird in that the left sidebar has dates, but if you look in the center div, the posts actually have titles. I shared two example titles above ^

I don't think consistency is worth it here. I think we should title them as whatever is interesting and eye-catching to the user. We should _try_ to think of a title that includes the major feature and some minor features. Hex's titles are great examples:

[Audit logs, expanded Git support, and Magic chart improvements](https://learn.hex.tech/changelog/2024-04-30)
[Reviews, data browser updates, and more](https://learn.hex.tech/changelog/2024-04-03)

Reply:
Oh, maybe a descriptive title could be:
> Two new extensions: unit and pgaudit